app/scripts/modules/core/src/serverGroup/ServerGroup.tsx

```diff
@@ -80,7 +80,7 @@ export class ServerGroup extends React.Component<IServerGroupProps, IServerGroup
 
       jenkins = {
         number: jenkinsConfig.number,
-        href: fromHost || fromFullUrl || fromBuildInfo,
+        href: fromBuildInfo || fromFullUrl || fromHost ,
```

Comment:
Seems like this should be `fromBuildInfo || fromHost || fromFullUrl`?

I honestly don't remember the nuance of when these various fields are populated and which takes precedence over which. Maybe it's fine as-is.

Reply:
I guess we can merge it and find out.